### ISSUE TITLE
Inline group by

### ISSUE THREAD

**Issue by Revadike** [DestinyItemManager/DIM#10152]
### Proposed change

I know I could ignore the `group weapons by` setting and put the grouping in the overall sorting, but then I don't get these helpful icons. I'm proposing to not start each group on a new line, but keep it all next to each other (inline) with the nice icon indicators separating the groups. This could also be an option.

### How does this fit into your workflow?

Better screen real estate.

**Comment by bhollis** [comment]
For reference that'd look like this:

<img width="905" alt="Screenshot 2023-12-05 at 10 58 44 PM" src="https://github.com/DestinyItemManager/DIM/assets/313208/8b3a7bed-13e8-4e15-a306-5ffc89525b8d">

To me it looks pretty messy, but I could see how it could still help folks find things.

**Comment by Shkvortex** [comment]
Seconding this request. I even made my own illustration by messing with the inspect tool before I saw it was already an open issue(I only added the bow icon though).
![image](https://github.com/DestinyItemManager/DIM/assets/26297400/0c6bf99f-934a-47b8-a7a0-d33820652ff6)

I think it keeps most of the benefit of grouping by giving your eye a consistent target to find, but it reduces the huge vertical impact of the grouping.
